access review policy (#12370)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accessreviewpolicy-update.md

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Update the properties of an [accessReviewPolicy](../resources/accessreviewpolicy.md) object.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|Policy.ReadWrite.AccessReviews|
|Delegated (personal Microsoft account)|Not supported.|
|Application|Policy.ReadWrite.AccessReviews|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
PATCH /policies/accessReviewPolicy
PATCH /identityGovernance/accessReviews/policy
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply a JSON representation of the [accessReviewPolicy](../resources/accessreviewpolicy.md) object.

The following table shows the properties that are required when you update the [accessReviewPolicy](../resources/accessreviewpolicy.md).

|Property|Type|Description|
|:---|:---|:---|

> |isGroupOwnerManagementEnabled|Boolean|If true, group owners can create and manage access reviews on groups they own|

Reviewer: ```suggestion
|isGroupOwnerManagementEnabled|Boolean|If `true`, group owners can create and manage access reviews on groups they own.|
```

## Response

> If successful, this method returns a `204 NoContent` response code.

Reviewer: ```suggestion
If successful, this method returns a `204 No Content` response code.
```

## Examples

### Request
<!-- {
  "blockType": "request",
  "name": "update_accessreviewpolicy"
}
-->
``` http
PATCH https://graph.microsoft.com/beta/policies/accessReviewPolicy
Content-Type: application/json

> Content-length: 163

Reviewer 2: Content-length isn't needed in the examples.

{
  "isGroupOwnerManagementEnabled": true
}
```

### Response
<!-- {
  "blockType": "response",
  "truncated": true
}
-->
``` http

> HTTP/1.1 204 NoContent

Reviewer: ```suggestion
HTTP/1.1 204 No Content
```

```

### Request
<!-- {
  "blockType": "request",
  "name": "update_accessreviewpolicy"
}
-->
``` http
PATCH https://graph.microsoft.com/beta/identityGovernance/accessReviews/policy
Content-Type: application/json
Content-length: 163

{
  "isGroupOwnerManagementEnabled": true
}
```

### Response
<!-- {
  "blockType": "response",
  "truncated": true
}
-->
``` http

> HTTP/1.1 204 NoContent

Reviewer: ```suggestion
HTTP/1.1 204 No Content
```
